sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
 ## Introduction
 
-### Potential writing prompt
+Biology and medicine are rapidly becoming particularly data-intensive field with
+respect to both research and practice. A recent comparison of genomics with
+social media, online videos and other data-intensive scientific disciplines
+suggested that the field of genomics alone would equal or surpass other fields
+in data generation and analysis within the next decade
+[@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
+also new challenges. We expect that algorithms to automatically extract
+meaningful patterns and provide sufficient context to enable us to act will be
+required.
 
-One potential future that we could imagine is a world in which data, once
-gathered, is rapidly integrated into our understanding of living systems. What
-we learn is rapidly put to use. Our health-related activities are constantly
-monitored (e.g. by wearables) and all of our interactions with health care
-systems are extensively tracked. These sources of information are combined to
-help to guide our health care and maintenance. We'd be able to compare our state
-and trajectory to (anonymized) others, and identify means to improve our health.
-These means might contain drug combinations selected based on personalized
-predictions.
+Concurrent with this explosive growth in biomedical data, a new class of machine
+learning algorithm has become widespread in the domain of image analysis.
+Computer scientists are now building many-layered neural networks from
+collections of millions of images. In a famous example, scientists from Google
+demonstrated that a neural network could learn to identify cats simply by
+watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
```

Comment:
The original reference to the Google cat study should actually be the following: 

Building high-level features using large scale unsupervised learning. ICML'12.  http://research.google.com/archive/unsupervised_icml2012.html

Unfortunately these conferences do not have DOI.